README.md

```diff
@@ -1 +1,30 @@
-This repository will contain sample code for interacting with XenServer
+# XenServer SDK usage examples
+
+This repository contains sample code for interacting with XenServer using the
+XenServer SDK.
+
+## Available languages
+
+There are examples for each of the five programming or scripting languages of
+the Xenserver SDK:
+
+- [C](c)
```

Comment:
maybe we should add link that redirect to the SDKs once we have it all figured out. i.e. when/if we publish them. Python already has this as it's built together with xapi and so the package is available in the releases section of the repository

Reply:
Definitely. This is the initial commit, we'll need to expand it in future. The READMEs per language as well will need better instructions on how to get the binaries etc,